dask/array/_shuffle.py

```diff
@@ -79,7 +87,44 @@ def shuffle(x, indexer: list[list[int]], axis):
     return Array(graph, out_name, chunks, meta=x)
 
 
-def _shuffle(chunks, indexer, axis, in_name, out_name, token):
+def _resize_other_dimensions(x: Array, longest_group, axis, chunks):
+    assert chunks == "auto", "Only auto is supported for now"
+    chunksize_tolerance = config.get("array.shuffle.chunksize-tolerance")
```

Comment:
could generalize this to things like `reshape` too?

Reply:
That sounds like a good idea, yes. I'll rename the option in another PR